docs/charts/index.yaml

```diff
@@ -2,68 +2,68 @@ apiVersion: v1
 entries:
   weblogic-operator:
   - apiVersion: v1
-    created: 2020-03-21T22:03:53.042035124Z
+    created: 2020-05-14T14:50:06.509204738-04:00
```

Comment:
If you merge in latest from develop, do changes to this file and the tgz file go away? These shouldn't be part of your change.

Reply:
I merged in the starting version.